ResetPasswordBundle/src/tests/UnitTests/Model/AbstractModelUnitTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SymfonyCasts\Bundle\ResetPassword\tests\UnitTests\Model;
+
+use PHPUnit\Framework\TestCase;
+use SymfonyCasts\Bundle\ResetPassword\tests\Contract\ModelUnitTestInterface;
+
+/**
+ * @author  Jesse Rushlow <jr@rushlow.dev>
+ */
+abstract class AbstractModelUnitTest extends TestCase implements ModelUnitTestInterface
```

Comment:
This is clever, but I'd rather not have this class. Two reasons: (A) many of the tests are overkill imo - like writing a test that a method *exists* and has the right visibility and (B) it makes the test classes themselves harder to read - which is important, because having tests that are readable and describe the object's intended behavior is pretty great.

So, I'd much prefer to delete this base class and focus on testing just the important stuff (any methods with logic) on the individual classes.

Reply:
removed